app/views/docs/getting-started-for-server.phtml

```diff
@@ -146,10 +146,11 @@ $client = (new Client())
         <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
             <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.client import Client
 
-client = Client()
+client = (Client()
     .set_endpoint('https://[HOSTNAME_OR_IP]/v1') # Your API Endpoint
     .set_project('5df5acd0d48c2')                # Your project ID
     .set_key('919c2db5d4...a2a3346ad2')          # Your secret API key
```

Comment:
Good catch!

I think I personally would prefer these to be formatted like this to  stay concise:
```py
client = (Client()
  .set_endpoint('https://[HOSTNAME_OR_IP]/v1') # Your API Endpoint
  .set_project('5df5acd0d48c2')                # Your project ID
  .set_key('919c2db5d4...a2a3346ad2'))         # Your secret API key
  ```

Reply:
Yes that definitely looks better, I will change it to reflect your proposal

Reply:
Yes that definitely looks better, I will change it to reflect your proposal